src/components/pages/Navbar/Navbar.js

```diff
@@ -1,34 +1,56 @@
-import { Avatar, Button, Input } from 'antd';
-import React from 'react';
+import { Avatar } from 'antd';
+import React, { useState } from 'react';
 import './Navbar.css';
 import logo from '../Navbar/ud_logo2.png';
+import { UserOutlined } from '@ant-design/icons';
+import { Layout } from 'antd';
+
+const { Header } = Layout;
 
 const Navbar = () => {
+  //when profile pictures are implemented into the backend, flesh out functions that will retrieve picture and set it to state
+
+  const [profilePic, setProfilePic] = useState(
```

Comment:
We can disregard the linter " 'setProfilePic' is assigned a value but never used " because we'll eventually use that to update the profile pic when we implement Redux.